lib/matplotlib/backends/web_backend/nbagg_mpl.js

```diff
@@ -123,6 +123,25 @@ mpl.figure.prototype._init_toolbar = function() {
     titlebar.prepend(buttongrp);
 }
 
+
+mpl.figure.prototype._canvas_extra_style = function(e){
+    // this is important to make the div 'focusable
+    e.attr('tabindex', 0)
+    // reach out to IPython and tell the keyboard manager to turn it's self
+    // off when our div gets focus
+
+    // location in version 3
+    if (typeof IPython.notebook.keyboard_manager != 'undefined') {
```

Comment:
I think the canonical way of doing this is `if (IPython.notebook.keyboard_manager)`.